components/Button/index.js

```diff
@@ -1,46 +1,53 @@
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 
 export const StyledButton = styled.button`
   border: 2px solid #dcae1d;
   border-radius: 10px;
   padding: 10px;
-  margin: 10px;
   color: #12343b;
   font-size: 12px;
   margin-bottom: 8px;
 `;
-
+export const SaveButton = styled(StyledButton)`
+  display: flex;
+  position: fixed;
+  margin: 0;
+  border: 2px solid black;
+  background-color: lightgreen;
+  bottom: 70px;
+  right: calc(50% - 190px);
+  padding: 0.7rem 1rem;
+  border-radius: 0.6rem;
+  color: black;
+  font-weight: bold;
+`;
```

Comment:
```suggestion
export const SaveButton = styled(StyledButton)`
  margin: 0;
  border: 2px solid black;
  background-color: lightgreen;
  padding: 0.7rem 1rem;
  border-radius: 0.6rem;
  color: black;
  font-weight: bold;
`;
```
Here's my alternative suggestion (!), and here's why:
I think the save button should be part of the form, semantically as well as visually, because that's where those buttons "belong". If you absolutely want to move the buttons to the bottom of the page, I probably wouldn't force them to appear there with CSS. 🤔

Reply:
Sidenote: I think the cancel button should be part of the `Button` component, as well. 👍

Reply:
Thanks for the review and for the suggestions.🙂